charts/library/common/templates/addons/code-server/_addon.tpl

```diff
@@ -0,0 +1,40 @@
+{{/*
+Template to render code-server addon
+It will include / inject the required templates based on the given values.
+*/}}
+{{- define "bjw-s.common.addon.codeserver" -}}
```

Comment:
should this be listed as a breaking change, since its bjw-s not common directly? (i'm fuzzy on helm templates still)

Reply:
That is a _very_ good point that I didn't even realise up until now. I've been doing this refactor under the assumption that folks are not really using the individual "functions". I'll blanket-mention the new "namespacing" as an explicit change in the changelog / PR.

Reply:
I'm just thinking of the authentik helm chart, which overrided some of the templates (in really weird ways) and called the helpers directly. I didn't check which helpers though.

Reply:
Looks like they are still on the k8s-at-home version, so in that sense this change shouldn't affect them. But if they want to move to the new maintained version they'll have to figure out how to migrate. (I have a connect to them through the k8s-at-home Discord, so I should be able to support them if required).

For now, having the warning in the PR description and the chart changelog annotations should be enough warning for anyone using the templates directly.